### ISSUE TITLE
Option to set fixed reservation ranges for reservations

### ISSUE THREAD

**Issue by rask** [rask/wprsrv#7]
Some reservables should only be able to be reserved for certain strict periods of time. Examples:
- A vacation house can be reserved in 1 week batches, no more and no less.
- A hotel room must be reserved for a two-day period.

If this feature is not implemented, it is up to the reservable author to set the reservation to use single-day reservations and block off dates manually (not wanted).

**Comment by novuu** [comment]
i suggest that strict periods of times should be set on top of reservables, similar to discounts/offers, by blocking n units of a default reservable (that's bookable for n time units).

**Comment by rask** [comment]
Do you mean a certain type of custom post type (of sorts) where users can create the time blocks and assign more extensive meta to each type of reservation block? And then each block could be assigned to reservable items however wanted?

**Comment by novuu** [comment]
yes, either some custom post type (which would allow assigning a reservable as needed) or if possible, just the taxonomy (which in essence allows the same, it would just require a new taxonomy to be created).
